app/javascript/react/screens/App/Plan/PlanConstants.js

```diff
@@ -73,4 +73,21 @@ STATUS_MESSAGES[STATUS_MESSAGE_KEYS.VM_MIGRATED] = __('Virtual machine migrated'
 STATUS_MESSAGES[STATUS_MESSAGE_KEYS.VM_TRANSFORMATIONS_FAILED] = __('VM migrations failed');
 STATUS_MESSAGES[STATUS_MESSAGE_KEYS.CANCELLED] = __('VM cancelled');
 
+const DOWNLOAD_LOG_STATUS_MESSAGES = {};
+DOWNLOAD_LOG_STATUS_MESSAGES.DOWNLOAD_LOG_ERROR_NO_HOST = __('Conversion host was not found');
+DOWNLOAD_LOG_STATUS_MESSAGES.DOWNLOAD_LOG_ERROR_NO_CREDS_SET = __('Credentials not set on conversion host');
+DOWNLOAD_LOG_STATUS_MESSAGES.DOWNLOAD_LOG_ERROR_SCP_ISSUE = __('Log file was not found on conversion host');
+DOWNLOAD_LOG_STATUS_MESSAGES.DOWNLOAD_LOG_ERROR_UNKNOWN_ISSUE = __('Unknown error');
```

Comment:
Since there is no corresp property in `BACKEND_ERROR_MESSAGES`, this message will never be seen, correct?

Reply:
Correct.

I intended on using it initially, but then figured it would be better to just return the original unparsed/unprocessed message instead.
Referring to https://github.com/ManageIQ/manageiq-v2v/pull/793/files#diff-0405502a3437597af99611db403b6a76R136

If it's causing confusion, we could remove it.

Reply:
I def agree with the decision to display the unprocessed message for the unknown cases

I have to admit that it did confuse me initially, seeing `return complexErrorMessage` as well as this property.  Can we remove it for now?  We can always bring it back if it becomes necessary

Reply:
Ok, no problem. Done.